extensions/exa-search/src/exa-search.tsx

```diff
@@ -0,0 +1,166 @@
+import { Action, ActionPanel, List, getPreferenceValues, showToast, Toast, Icon } from "@raycast/api";
+import React from "react";
+import fetch from "node-fetch";
+
+interface Preferences {
+  exaApiKey: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x

Reply:
Thanks for the feedback, I addressed all of it. 
- I generated an icon
- Moved the screenshot referenced in the README to a media directory
- Followed the docs you referenced for the screenshots, they are in the metadata directory now
- Updated the `@raycast/api` to the latest version.